common/__generated__/graphql.ts

```diff
@@ -388,11 +390,13 @@ export enum ActionDateFormat {
 export type ActionDescriptionBlock = FieldBlockMetaInterface & StreamFieldInterface & {
   __typename?: 'ActionDescriptionBlock';
   blockType: Scalars['String'];
+  blocks: Array<StreamFieldInterface>;
   field: Scalars['String'];
+  fieldHelpText?: Maybe<Scalars['String']>;
+  fieldLabel?: Maybe<Scalars['String']>;
```

Comment:
Is it intentional that these new fields exist on all action blocks? And that `blocks` has been added and `value` removed?

Reply:
Yes, sorry I should have mentioned the related changes in the backend. The fields are intentional, as we decided to add support for these customizable fields for all dynamically created blocks of type `ActionListContentBlock`, see [here](https://github.com/kausaltech/kausal-watch-private/pull/138).

The removal of value and addition of blocks is because of the change from Staticblocks to Structblocks to support the added fields.